syne_tune/backend/local_backend.py

```diff
@@ -54,9 +55,11 @@ def __init__(
             scheduled on a different GPU. A new trial is preferentially
             scheduled on a free GPU, and otherwise the GPU with least prior
             assignments is chosen
+        :param delete_checkpoints: If True, checkpoints of stopped or completed
```

Comment:
Currently with the modification proposed, the checkpoints are stopped when we stop, not just when they are completed. This is a good approach (cleaning them whenever stop happens) so I would be keen on keeping this behavior, we just need to update the doc to "If True, checkpoints folders of stopped trial are deleted."

Reply:
Maybe I am wrong, but I thought if a trial ends on its own, it gets status completed but not stopped, right?

But even if a trial that gets stopped, is later changed to completed, that is fine, because delete_checkpoint can be called several times. It just does not do anything later on.